amplipi/models.py

```diff
@@ -996,7 +997,8 @@ class Config:
                 "git_dirty": False,
               }
             ],
-            'stream_types_available': ['bluetooth', 'fmradio']
+            'stream_types_available': ['bluetooth', 'fmradio'],
+            'connected_drives': [],
```

Comment:
might be helpful to show something here. from this alone, it's unclear if it will show the mountpoint, or the device itself (i suspect i know the answer, but that's not clear.)